apps/ledger-live-desktop/src/renderer/modals/Send/AccountFooter.tsx

```diff
@@ -23,7 +24,13 @@ const AccountFooter = ({ account, parentAccount, status }: Props) => {
   const mainAccount = getMainAccount(account, parentAccount);
   const feesCurrency = getFeesCurrency(mainAccount);
   const feesUnit = getFeesUnit(feesCurrency);
-  return (
+
+  const cryptoCurrency = mainAccount.currency;
+  const specific = cryptoCurrency ? getLLDCoinFamily(cryptoCurrency.family) : null;
+  const SpecificComponent = specific?.AccountFooter;
+  return SpecificComponent ? (
+    <SpecificComponent account={account} parentAccount={parentAccount} status={status} />
```

Comment:
could you try to cover it on the playwright tests? like doing a Send flow with Ethereum would do? 🤔

Reply:
Yeah sure I can try to add one